benches/benchmarks/check_texts.rs

```diff
@@ -29,12 +31,12 @@ async fn request_until_success(req: &Request, client: &ServerClient) -> Response
 }
 
 #[tokio::main]
-async fn check_text_basic(text: &str) -> Response {
+async fn check_text_basic(text: &'static str) -> Response {
```

Comment:
Is there any reason for this to be static?

Reply:
This was only static because of `with_text` and `with_data` - I'll fix that